subprojects/packagefiles/libgrapheme/gen/meson.build

```
@@ -24,6 +24,13 @@ gen_headers_tgt = {
 }
 
 chdir_wrapper = files('chdir_wrapper.py')
```

You should be able to undo all of these changes (including the removal of the `#!` line) and just change this to
```suggestion
chdir_wrapper = find_program('chdir_wrapper.py')
```

Meson has a special case for scripts starting with `#!/usr/bin/env python3`, so execute permission shouldn't be needed.  However, with `custom_target()` this works for `find_program()` but not for `files()`.